From be5f79ab7116730967d5d426880706d9d4cf2289 Mon Sep 17 00:00:00 2001
From: Chandrakala Chavva <Chandrakala.Chavva@cavium.com>
Date: Sat, 8 May 2021 15:21:39 -0700
Subject: [PATCH 1524/1921] driver: soc: marvell: Don't enable mvmdio driver by
 default

Signed-off-by: Chandrakala Chavva <Chandrakala.Chavva@cavium.com>
Change-Id: I7c11dcd401698d453dd72d9ff12fc795b17847a1
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/51353
Reviewed-by: Damian Eppel <deppel@marvell.com>
Reviewed-by: Chandrakala Chavva <cchavva@marvell.com>
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/soc/marvell/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/soc/marvell/Kconfig b/drivers/soc/marvell/Kconfig
index 4a6d4e377208..2df85dbaeee6 100644
--- a/drivers/soc/marvell/Kconfig
+++ b/drivers/soc/marvell/Kconfig
@@ -123,8 +123,8 @@ config OCTEONTX_INFO
 
 config MVMDIO_UIO
 	tristate "Marvell MDIO interface driver"
-	  default m
-	select MDIO_THUNDER
+	default m
+	depends on MDIO_THUNDER
 	help
 	  Select this option if you want to provide an interface for userspace
 	  PHY drivers to access Marvell's MDIO bus.
-- 
2.31.1

